scripts/get_visualization.py

```diff
@@ -0,0 +1,84 @@
+"""Creates a collection of graphs summarizing results saved to a directory."""
+import argparse
+from typing import Dict, Tuple
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+CUSTOM_APPROACHES = ["llm-standard-plan", "llm-multi", "llm-multi-plan"]
+BASELINE_APPROACHES = ["fd", "pyperplan"]
+
+
+def _main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input_path", type=str)
+    parser.add_argument("--output_path", type=str)
+    args = parser.parse_args()
+    custom_approaches, baseline_approaches = _construct_dictionaries(
+        args.input_path)
+    _get_visualization(custom_approaches, baseline_approaches,
+                       args.output_path)
+
+
+def _construct_dictionaries(input_path: str) -> Tuple[Dict, Dict]:
+    custom_approaches: Dict = {}
+    for name in CUSTOM_APPROACHES:
+        custom_approaches[name] = {}
+    baseline_approaches: Dict = {}
+    for name in BASELINE_APPROACHES:
+        baseline_approaches[name] = {}
+
+    df = pd.read_csv(input_path)
+    column_labels = list(df.columns)
+    env_column_index = column_labels.index('env')
+    approach_column_index = column_labels.index('approach')
+    accuracy_column_index = column_labels.index('success')
+    experiment_name_index = column_labels.index('experiment_id')
+
+    for _, row_raw in df.iterrows():
+        row = row_raw.tolist()
+        approach_name = row[approach_column_index]
+        environment_name = row[env_column_index]
+        accuracy = row[accuracy_column_index]
+        accuracy = float(accuracy[0:accuracy.index(' ')])
+        experiment_name = row[experiment_name_index]
+        for approach in baseline_approaches:
+            if approach_name == "pure-planning" and approach in experiment_name:
```

Comment:
if we add a baseline other than pure planning, we'll have to change this -- for now, you can just add a comment here reminding us